test/unit/models/embassy_test.rb

```diff
@@ -1,49 +1,12 @@
 require "test_helper"
 
 class EmbassyTest < ActiveSupport::TestCase
-  setup do
-    @location = create(:world_location, :with_worldwide_organisations, name: "Narnia")
-    @organisation = @location.worldwide_organisations.first
-
-    @organisation.main_office = create(
-      :worldwide_office,
-      title: "British Embassy Narnia",
-      worldwide_organisation: @organisation,
-      worldwide_office_type: WorldwideOfficeType::Embassy,
-      contact: create(:contact, street_address: "The woods", country: @location),
-    )
-  end
-
   test "it delegates to the world location" do
-    embassy = Embassy.new(@location)
+    location = create(:world_location, :with_worldwide_organisations, name: "Narnia")
+    embassy = Embassy.new(location)
     assert_equal "Narnia", embassy.name
   end
 
-  test "remote_services_office and remote_services_country" do
```

Comment:
Minor: Can you explain in the commit note *why* it's OK to remove these tests?

Reply:
I've now added additional explanation in the commit message and force-pushed.